example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/LaunchLog.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.reportportal.example.cucumber.logging;
+
+import ch.qos.logback.classic.Level;
+import com.epam.reportportal.example.cucumber.util.Attachment;
+import com.epam.reportportal.service.ReportPortal;
+import cucumber.api.java.en.Given;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Random;
+
+import static com.epam.reportportal.example.cucumber.util.AttachmentHelper.FILE_NAME;
+import static com.epam.reportportal.example.cucumber.util.AttachmentHelper.getFileFromResources;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class LaunchLog {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(LaunchLog.class);
+
+	@Given("I attach files to launch logs")
+	public void reportStep() {
+		Arrays.stream(Attachment.values()).forEach(it -> launchLogWithAttachment(randomLogLevel(), FILE_NAME, it));
+		LOGGER.info("Add logs with attachments to launch");
```

Comment:
I think this line should be before attachment logging.

Reply:
fixed